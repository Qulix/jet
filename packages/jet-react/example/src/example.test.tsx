import React from 'react';
import { shallow } from 'enzyme';

const Text = () => <p>Text</p>;

describe('Example', () => {

  it('should match snapshot', () => {
    const wrapper = shallow(<Text />);
    expect(wrapper).toMatchSnapshot('<Text /> Snapshot');
  });

})
