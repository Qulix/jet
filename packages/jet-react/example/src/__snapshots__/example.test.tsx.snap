// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Example should match snapshot: <Text /> Snapshot 1`] = `
<p>
  Text
</p>
`;

exports[`Example should math snapshot: <Text /> Snapshot 1`] = `
<p>
  Text
</p>
`;
